### PR TITLE
Move some e2e-agnostic-upgrade presubmits to Azure

### PR DESCRIPTION
**Pull request by vrutkovs**
Moving some tests from GCP to Azure, as AWS and GCP are over capacity - and Azure upgrade tests have high pass rate

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]
/hold

We may want to see the rehearsals

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml [hunk 1]
```diff
@@ -65,8 +65,8 @@ tests:
     workflow: openshift-e2e-gcp
 - as: e2e-gcp-upgrade
   steps:
-    cluster_profile: gcp
-    workflow: openshift-upgrade-gcp
+    cluster_profile: azure4
```
Comment (petr-muller) [inline ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:68]: Huh mmnt, this one is not agnostic

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-release-4.7.yaml [hunk 2]
```diff
@@ -57,8 +57,8 @@ tests:
     workflow: openshift-e2e-gcp
 - as: e2e-gcp-upgrade
   steps:
-    cluster_profile: gcp
-    workflow: openshift-upgrade-gcp
+    cluster_profile: azure4
+    workflow: openshift-upgrade-azure
```
Comment (petr-muller) [inline ci-operator/config/openshift/insights-operator/openshift-insights-operator-release-4.7.yaml:61]: THis one too

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml (hunk 1 above)
Comment (petr-muller) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:68]: We may as well bite the bullet and rename it, if this is intentional?

Comment (vrutkovs) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:68]: Right, let me do this

Comment (vrutkovs) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:68]: Done

**Review by petr-muller — APPROVED** [review]
:thisisfine:

**Comment by vrutkovs** [comment]
/hold cancel

rehearsals look fine

**Pull request merged**